src/main/java/utility/CheckValueCorrect.java

```diff
@@ -0,0 +1,44 @@
+package utility;
+
+public class CheckValueCorrect {
+    public String value;
+
+    public CheckValueCorrect(){}
+    public CheckValueCorrect(String inputValue) {
+        this.value = inputValue;
+    }
+
+    public int strToInt(String inputValue){
+        this.value = inputValue;
+        int intValue;
+
+        try {
+            intValue = Integer.parseInt(this.value);
+        }catch (Exception e){
+            throw new IllegalArgumentException("-".repeat(58)+"\n" +
+                    "정수로 변환할 수 없는 값이 감지되었습니다. 아래의 형식에 맞게 입력해 주세요.\n" +
+                    "➣ 커스텀 구분자 지정시, //와 \\n를 모두 사용하며 순서를 지켜야 합니다.\n" +
+                    "➣ 구분자를 제외한 모든 문자는 양수인 정수여야 합니다.\n");
+        }
+
+        if(intValue < 0) {
```

Comment:
이렇게 되면  0일때는 검사를 못할것 같아요!
```suggestion
        if(intValue <= 0) {
```
이렇게 수정하는게 좋을 것 같아요!